### PR TITLE
Fix: pfe-codeblock | Update theme colours for accessibility

### LINKED ISSUE #1410: pfe-codeblock | update theme colors for accessibility
## Description of the issue

Some of the colours in the code block do not meet the accessibility standard please update so colour contrast meets the standard.

### Impacted component(s)

- pfe-codeblock

### Steps to reproduce

Run any accessibility tool against the component demo page.

### Expected behavior

Colour contrast tests pass

Colours should be as follows:

### Light theme

$custom-pink: #B300B3; // `h1, p, body`
$custom-red: #C9190B; // `"javascript", red, #eee`
~~$custom-orange: #EC861B;~~ // `Use $custom-pink`
$custom-blue: #06C; // `<, </, >, height=""`
~~$custom-teal: #39ADB5;~~ // `Use $custom-blue`
$custom-purple: #40199A; // `const, let, null, main, printf, return`
$custom-white: #F5F5F5; // Background
$custom-gray: #6A6E73; // `example, text, #include, stdio.h, int`
$custom-light-gray: #6A6E73; // Numbers and dividers
$custom-black: #151515; // `example html, some paragraph text`
Font: Consolas / 16pt / 24 (1.5)

https://xd.adobe.com/view/166cbba5-d3c9-4068-938e-6941c26c39c9-433f/specs/

### Dark theme

$custom-red: #FF8A80; // `h1, p, body`
~~$custom-orange: #FE9A8C; //(red, #eee, etc.)~~ // `Use $custom-green`
~~$custom-yellow: #FFCB6B; //(height, width, cx, cy, etc.)~~ // `Use $custom-blue`
$custom-green: #ACE12E; // `"javascript", red, #eee`
$custom-blue: #73BCF7; // `<, </, >, height=""`
~~$custom-teal: #80CBC4;~~ // `Use $custom-blue`
$custom-purple: #B2A3FF; // `const, let, null, main, printf, return`
$custom-black: #3C3F42; // Background
$custom-gray: #D2D2D2; // `example, text, #include, stdio.h, int`
$custom-dark-gray: #D2D2D2; // Numbers and dividers
$custom-white: #FFF; // `example html, some paragraph text`
Font: Consolas / 16pt / 24 (1.5)

$custom-hilight-bg: #363636; // Not sure where this maps

https://xd.adobe.com/view/166cbba5-d3c9-4068-938e-6941c26c39c9-433f/screen/8117f58c-c2ad-4156-b52f-d0c6e21cc27a/specs/

### PR DESCRIPTION
**Pull request by alistairmc**
## pfe-codeblock | update theme colors for accessibility

Updating the theme colours for accessibility as described in the issue #14107

### Related issues

Closes #1410
- (#1410) pfe-codeblock | update theme colors for accessibility

### Preview

Link(s) to demo page(s) where this element can be viewed:
- [Link](https://deploy-preview-1418--happy-galileo-ea79c4.netlify.app/elements/pfe-codeblock/demo/) 

### What has changed and why

- Updated light and dark theme colours to meet accessibility requirements.

Colours should be as follows:

Light theme
$custom-pink: #B300B3; // h1, p, body
$custom-red: #C9190B; // "javascript", red, #eee
$custom-orange: #EC861B; // Remove and replace with  $custom-pink
$custom-blue: #06C; // <, </, >, height=""
$custom-teal: #39ADB5; // Remove and replace with  $custom-blue
$custom-purple: #40199A; // const, let, null, main, printf, return
$custom-white: #F5F5F5; // Background
$custom-gray: #6A6E73; // example, text, #include, stdio.h, int
$custom-light-gray: #6A6E73; // Numbers and dividers
$custom-black: #151515; // example html, some paragraph text

Dark theme
$custom-red: #FF8A80; // h1, p, body
$custom-orange: #FE9A8C; // Remove and replace with $custom-green
$custom-yellow: #FFCB6B; // Remove and replace with  $custom-blue
$custom-green: #ACE12E; // "javascript", red, #eee
$custom-blue: #73BCF7; // <, </, >, height="", height, width, cx, cy, etc.
$custom-teal: #80CBC4; // Remove and replace with  $custom-blue
$custom-purple: #B2A3FF; // const, let, null, main, printf, return
$custom-black: #3C3F42; // Background
$custom-gray: #D2D2D2; // example, text, #include, stdio.h, int
$custom-dark-gray: #D2D2D2; // Numbers and dividers
$custom-white: #FFF; // example html, some paragraph text

### Testing instructions

1. go to the pfe-codeblock demo page and run a accessibility contract checker.

#### Browser requirements

Your component should work in all of the following environments:

- [ ] Latest 2 versions of Edge
- [ ] Internet Explorer 11 (should be useable, not pixel perfect)
- [x] Latest 2 versions of Firefox (one on Mac OS, one of Windows OS)
- [ ] Firefox 68 (or latest version for Red Hat Enterprise Linux distribution)
- [x] Latest 2 versions of Chrome (one on Mac OS, one of Windows OS)
- [x] Latest 2 versions of Safari
- [ ] Android mobile device (such as the Galaxy S9)
- [x] Apple mobile device (such as the iPhone X)
- [x] Apple tablet device (such as the iPhone Pro)

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [x] Tests have been updated to cover these changes.
- [x] Browser testing passed.
- [x] Repository compiles and tests pass.
- [x] Changelog updated.
- [ ] Documentation (README.md, WHY.md, etc.) updated or added.
- [ ] Link to the demo recording: []()
- [x] Approved by designer.

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

**Comment by alistairmc** [comment]
@coreyvickery I have made the colour changes requested in this PR, please review and let me know if its ok?

**Comment by alistairmc** [comment]
see comment on linked issue font is fine re @coreyvickery Overpass is fine for now and 1em (16px) is fine for all sizes too.

**Comment by alistairmc** [comment]
@castastrophe there seems to be an error running visual regression its getting the following error:- 
Error: Action Failed: Error: Action input failed for reason: BROWSERSTACK_ACCESS_KEY not found. Use 'browserstack/github-actions/setup-env@master' Action to set up the environment variables before invoking this Action

Have you seen this before, do you know what I need to update?

**Comment by castastrophe** [comment]
@alistairmc Ah, I didn't notice this is a fork branch.  The fork doesn't have access to the shared secrets on the project so Browserstack can't access the login credentials.

**Comment by castastrophe** [comment]
@alistairmc If you have BrowserStack credentials, you can add them to a .env file in your project locally (it will not be committed because it's an ignored file):

```
BROWSERSTACK_USER=<username>
BROWSERSTACK_KEY=<key>
```

Then locally you can run: `npm run e2e pfe-codeblock` and check the test/vrt-snapshots folder for the results.

**Comment by alistairmc** [comment]
> @alistairmc Ah, I didn't notice this is a fork branch. The fork doesn't have access to the shared secrets on the project so Browserstack can't access the login credentials.

@castastrophe Do I need to create a branch on this repo to get the visual regression to run, I have tested it locally and the e2e passes.

**Comment by castastrophe** [comment]
@alistairmc I see the updated baselines now so that looks great!  Just needs a changelog update and it's good from my perspective!

**Comment by alistairmc** [comment]
@castastrophe what is or process for changeling generation is it automated with something like generate-changelog npm? or is it a manual process?

**Comment by castastrophe** [comment]
@alistairmc Thanks for asking - yes changelog updates are manual.  You can add your entry in the 1.3.3 release heading using the following format:

```
- [](https://github.com/patternfly/patternfly-elements/commit/) fix: pfe-codeblock - update theme colours for accessibility
```

File: CHANGELOG-1.x.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # 1.X (TBD)
 
 - [06257ba](https://github.com/patternfly/patternfly-elements/commit/06257ba7edc3ca70ddfd0dd6c71e600540ac692b) fix: pfe-accordion accessibility issues
-
+- [fb11fba](https://github.com/patternfly/patternfly-elements/commit/fb11fba1932cde8f6f6d1632000233f0394d9c1c) fix: pfe-codeblock - update theme colours for accessibility
```
Comment (castastrophe) [inline CHANGELOG-1.x.md:4]:
```suggestion
- [](https://github.com/patternfly/patternfly-elements/commit/) fix: pfe-codeblock - update theme colours for accessibility
```

It's okay to leave this blank - we won't know the commit hash until it's merged with master

**Comment by castastrophe** [comment]
This is looking great - if Corey has approved and it's been tested in our supported browsers, I think it's good to merge.  I added a small note to the changelog but 👍

**Review by castastrophe — APPROVED** [review]

**Comment by coreyvickery** [comment]
@castastrophe @alistairmc Looks good to me.

**Pull request merged**